site_scons/compiler_setup.py

```diff
@@ -59,6 +59,14 @@ def base_setup(env, prereqs=None):
             env.AppendUnique(CCFLAGS=['-Og'])
         else:
             env.AppendUnique(CCFLAGS=['-O0'])
+    if build_type == 'lcov':
+        if compiler not in ('gcc', 'mpicc'):
+            print("lcov builds only work with gcc '{}'".format(compiler))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string